apps/www/registry/default/ui/carousel.tsx

```diff
@@ -211,6 +211,7 @@ const CarouselPrevious = React.forwardRef<
       )}
       disabled={!canScrollPrev}
       onClick={scrollPrev}
+      style={{ touchAction: "manipulation", ...props.style }}
```

Comment:
@mukul-turing is there a classname we can use instead of adding the CSS using the style property?

Maybe [like this](https://github.com/huntabyte/shadcn-svelte/pull/654/files#diff-76c7c20f4c1cffbe752d54b93a687df80adb73e2e62de3b5670a9f83a748abd1R22) example?

Reply:
Thanks @davidjerleke for the feedback, I will update the PR.

Reply:
Thanks for the swift response @mukul-turing. I’ll resolve my comments 👍🏻.